library-app/src/helpers/roles.tsx

```diff
@@ -0,0 +1,23 @@
+import { Jwt, roleKey, UserRole } from '../models/jwt.model';
+// eslint-disable-next-line camelcase
```

Comment:
Maybe you can solve this by typing import jwt_decode as JwtDecode from 'jwt-decode'; but leave it like this for now